lib/foreman_tasks_core/callback_action.rb

```diff
@@ -0,0 +1,11 @@
+module ForemanTasksCore
+  class CallbackAction < ::Dynflow::Action
+    def plan(callback, data)
+      plan_self(:callback => callback, :data => data)
+    end
+
+    def run
+      SmartProxyDynflowCore::Callback::Request.send_to_foreman_tasks(input[:callback], input[:data])
```

Comment:
Just wondering if this is the right place for this to be in? I know this will probably be only used from within `SmartProxyDynflowCore`, but `ForemanTasks{,Core}` doesn't have any kind of dependency on SPDC.